packages/battery_plus/battery_plus/example/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
This change is an auto migration from a more recent version of flutter. Happy to remove if you want.